src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/config/CoroutineScopes.kt

```diff
@@ -0,0 +1,12 @@
+package uk.gov.justice.digital.hmpps.prisonertonomisupdate.config
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class CoroutineScopes {
+  @Bean
+  fun reportScope() = CoroutineScope(Dispatchers.Unconfined)
+}
```

Comment:
everything we do is non-blocking so we don't care about sticking to a particular thread